### PR TITLE
64 my pets view

### LINKED ISSUE #64: my-pets View

### PR DESCRIPTION
**Pull request by ayeza-zamudio3**
# Description
end style view my pets

### CODE REVIEW

**Review by daniel-yocupicio — DISMISSED** [review]

File: src/assets/pets.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "id": 1,
+        "image": "https://cdn.mequieroir.com/wp-content/uploads/2019/10/mascota-espana-blanca-1024x639.jpg",
+        "name": "Chino",
+        "male": "Male",
+        "color": "Color",
+        "age": "Age",
+        "characteristics": "Characteristics"
+      }
+    ]
+}
```
Comment (YilKanda) [inline src/assets/pets.json]: Add a blank line

File: src/screens/index.js [hunk 2]
```diff
@@ -5,4 +5,5 @@ export {default as Home} from './Home/HomeScreen';
 export {default as ResetPassword} from './ResetPasswordScreen/ResetPasswordScreen';
 export {default as SignInScreen} from './SingInScreen/SignInScreen';
 export {default as HomeClient} from './Client/Home/HomeScreen';
-export {default as PetFormScreen} from './Client/PetFormScreen/PetFormScreen';
\ No newline at end of file
+export {default as PetFormScreen} from './Client/PetFormScreen/PetFormScreen';
+export {default as Mypet} from './Pets/Mypet';
```
Comment (YilKanda) [inline src/screens/index.js]: Add a blank line

**Review by YilKanda — CHANGES_REQUESTED** [review]
Resolve the conflicts

imagen

**Review by GerardoTovar — DISMISSED** [review]

File: src/screens/index.js (hunk 2 above)
Comment (FranciscoSenestrari) [reply src/screens/index.js]: fix conversation

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by LizyMiGo — APPROVED** [review]

**Pull request merged**